exec/cnex/lib/random.h

```diff
@@ -0,0 +1,9 @@
+#ifndef RANDOM_H
+#define RANDOM_H
+#include <stdint.h>
```

Comment:
This include file does not appear to be needed.

Reply:
Removed.